src/hpp/utils.py

```diff
@@ -0,0 +1,30 @@
+# Copyright (c) 2020, CNRS
+# Authors: Guilhem Saurel <guilhem.saurel@laas.fr>
+
+import os
+import subprocess
+import time
+
+
+class ServerManager:
+    """A context to ensure a server is running."""
+    def __init__(self, server):
```

Comment:
Can't we have a default value to the argument ?

Reply:
excellent idea :)